package/osx/cmake/merge-electron.cmake

```diff
@@ -18,10 +18,11 @@ function(echo MESSAGE)
    execute_process(COMMAND echo "-- ${MESSAGE}")
 endfunction()
 
-# run yarn to ensure required packages are installed
+# run npm to ensure required packages are installed
 execute_process(
 	COMMAND
-		"@YARN@"
+		"@NPM@"
+		"install"
```

Comment:
Consider using `npm ci` instead of `npm install` in scripts. The `package-lock.json` file should be committed as part of the git history, so that `npm ci` can be used to guarantee that the packages installed are 100% the same across machines. The `ci` command means "clean-install," and only reads out of the `package-lock.json` file, which is faster and safer than performing dependency resolution with `npm install`. The `install` command is better for adding packages or updating minor versions of existing packages. This also prevents against a constantly-changing `package-lock.json` file with each build.

Reply:
Yup! Matheus gave same feedback on Slack (some threads about it in Electron channel).